docs/src/references/tedge-configuration-management.md

```diff
@@ -140,12 +140,12 @@ For example, it subscribes to the following topic for the main device.
 tedge mqtt sub 'te/device/main///cmd/config_snapshot/+'
 ```
 
-When a new config snapshot command with the ID "1234" is published by another component, it appears as follows.
+When a new config snapshot command with the ID "1234" for the device name "example" is published by another component, it appears as follows.
```

Comment:
```suggestion
When a new config snapshot command with the ID "1234" is published for the device named "example" by another component, it appears as follows.
```

I would even consider to fully rewrite the sentence.

To start a new config snapshot with the ID "1234" on the device named "example", a component has to publish the following message over MQTT: